wiki/Tournament_Management_Commands/en.md

```diff
@@ -53,4 +53,6 @@ The following is an example of command usage:
 - `!mp team Zallius blue` - Move Zallius to the blue team.
 - `!mp team Loctav red` - Move Loctav to the red team.
 - `!mp set 0 2` - Set the team mode to Head To Head and scoring mode to Combo.
-- `!mp start` - Start the match instantaneously.
\ No newline at end of file
+- `!mp start` - Start the match instantaneously.
+
+We expect you to act professionally and responsibly in the management of your tournament. Any actions with malicious intent will not be treated lightly.
```

Comment:
Swap 'actions with malicious intent line for':

Any abuse of these commands to disrupt the play of other users will be dealt with harshly as a violation of our community rules.